kfs-core/src/main/java/edu/arizona/kfs/fp/businessobject/ProcurementCardTransaction.java

```diff
@@ -0,0 +1,2014 @@
+package edu.arizona.kfs.fp.businessobject;
+
+import java.math.BigDecimal;
+import java.sql.Date;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+
+import org.apache.commons.beanutils.converters.SqlDateConverter;
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.kuali.rice.krad.bo.PersistableBusinessObjectBase;
+
+public class ProcurementCardTransaction extends PersistableBusinessObjectBase {
```

Comment:
ProcurementCardTransaction is the "load" table for the incoming PCard transactions from JPMC, hence the large number of fields that are later split across several different tables in the "create document" step of the batch job.